lms/djangoapps/verify_student/services.py

```diff
@@ -136,3 +137,25 @@ def get_attempts(self, user_id, course_id, related_assessment_location):
         """
         course_key = CourseKey.from_string(course_id)
         return VerificationStatus.get_user_attempts(user_id, course_key, related_assessment_location)
+
+    def get_course_verification_status(self, user, course_id):
```

Comment:
And it looks like the tests use `self.user.id`, so we're already calling it with ids?